vignettes/adrg.qmd

```diff
@@ -0,0 +1,785 @@
+
+: {tbl-colwidths="\[20,20,10,10,20,10,10\]"}
+
+### ADSL - Subject-Level Analysis Dataset
+
+The subject level analysis dataset (ADSL) contains required variables for demographics, treatment groups, and population flags. In addition, it contains other baseline characteristics that were used in both safety and efficacy analyses. All patients in DM were included in ADSL. The following are the key population flags are used in analyses for patients: • SAFFL – Safety Population Flag (all patients having received any study treatment) • ITTFL – Intent-to-Treat Population Flag (all randomized patients)
+
+### ADADAS - ADAS-COG Analysis Dataset
+
+ADADAS contains analysis data from the ADAS-Cog questionnaire, one of the primary efficacy endpoints. It contains one record per subject per parameter (ADAS-Cog questionnaire item) per VISIT. Visits are placed into analysis visits (represented by AVISIT and AVISITN) based on the date of the visit and the visit windows.
+
+### ADAE - Adverse Events Analysis Dataset
+
+ADAE contains one record per reported event per subject. Subjects who did not report any Adverse Events are not represented in this dataset. The data reference for ADAE is the SDTM AE (Adverse Events) domain and there is a 1-1 correspondence between records in the source and this analysis dataset. These records can be linked uniquely by STUDYID, USUBJID, and AESEQ. Events of particular interest (dermatologic) are captured in the customized query variable (CQ01NAM) in this dataset. Since ADAE is a source for ADTTE, the first chronological occurrence based on the start dates (and sequence numbers) of the treatment emergent dermatological events are flagged (AOCC01FL) to facilitate traceability between these two analysis datasets.
+
+### ADLBC - Analysis Dataset Lab Blood Chemistry
+
+ADLBC contains one record per lab analysis parameter, per time point, per subject. ADLBC contains lab chemistry parameters and these data are derived from the SDTM LB (Laboratory Tests) domain. Two sets of lab parameters exist in ADLBC. One set contains the standardised lab value from the LB domain and the second set contains change from previous visit relative to normal range values. In some of the summaries the derived end-of-treatment visit (AVISITN=99) is also presented.
+
+### ADTTE - AE Time To 1st Derm. Event Analysis
+
+ADTTE contains one observation per parameter per subject. ADTTE is specifically for safety analyses of the time to the first dermatologic adverse event. Dermatologic AEs are considered an adverse event of special interest. The key parameter used for the analysis of time to the first dermatological event is with PARAMCD of "TTDE".
+
+{{< pagebreak >}}
+
+# Data Conformance Summary
+
+## Conformance Inputs
+
+Were the analysis datasets evaluated for conformance with CDISC ADaM Validation Checks?
+
+```         
+  Yes, Version of CDISC ADaM Validation Checks and software used: Pinnacle 21® 
```

Comment:
With the re-submission - should get an updated P21 report just for timestamps alignment?

Reply:
Yes, this would be good to do. Might be good to do a rerun refresh of all the programs and outputs again for the re-submission, as well as do a P21 run. This action still open.